cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -96,12 +98,36 @@ func startMizarPodController(ctx *ControllerContext, grpcHost string, grpcAdapto
 	controllerName := "mizar-pod-controller"
 	klog.V(2).Infof("Starting %v", controllerName)
 
-	go controllers.NewMizarPodController(
-		ctx.InformerFactory.Core().V1().Pods(),
-		ctx.ClientBuilder.ClientOrDie(controllerName),
-		grpcHost,
-		grpcAdaptor,
-	).Run(mizarPodControllerWorkerCount, ctx.Stop)
+	//informerFactory is needed by arktosinformer.NetworkInformer.Lister(),
```

Comment:
nit - this block of comments seems not essential for understanding. suggest to remove.

Reply:
removed it.